ddpui/ddpairbyte/schema.py

```
@@ -1,8 +1,229 @@
+    """Schema for a full Airbyte source object."""
+
+    sourceId: str
+    sourceDefinitionId: str  # Renamed from sourceDefId to match common Airbyte API naming
+    workspaceId: str
+    name: str
+    sourceName: str
+    configuration: Dict[str, Any]  # 'config' in Airbyte API is 'configuration' in response
+    icon: Optional[str] = None
+
+
+class AirbyteConnectionSchemaUpdate(Schema):
```

Ruff (0.12.2)

Undefined name `Schema`

(F821)